src/tree.test.visuals.ts

```diff
@@ -63,13 +72,13 @@ for (const story of stories.Tree) {
           await page
             .locator('glide-core-tree-item-menu')
             .first()
-            .locator('glide-core-icon-button')
+            .getByRole('button')
```

Comment:
Slightly less brittle in that Tree can swap out Icon Button for any other button and the test will still pass.